crates/mipsy_web/src/components/data_segment.rs

```diff
@@ -145,8 +159,8 @@ fn render_page(props: &DataSegmentProps, page_addr: u32, page_contents: Vec<Safe
                         let full_offset = nth * ROW_SIZE + offset;
                         let full_page_addr = page_addr as usize + full_offset;
                         let class = "cursor-help";
-                        let mut style = "border: 2px solid ".to_string();
-                        let mut title = "".to_string();
+                        let mut style = String::new();
```

Comment:
style could still be initialised as `String::from("border: 2px solid ");` but that's up to you